src/dune_rules/cm_files.mli

```diff
@@ -15,8 +15,12 @@ val make :
   -> unit
   -> t
 
+val filter_excluded_modules : t -> Module.t list -> Module.t list
+
 val unsorted_objects_and_cms : t -> mode:Mode.t -> Path.t list
 
+val melange_objects_and_cms : t -> Path.t list
```

Comment:
Is this function being used anywhere?

Reply:
Nope, great catch, it was used for the cma hack, but I forgot to remove it in https://github.com/jchavarri/dune/pull/4/commits/7882f6ff3b30aea09008c208a4a3f67d33bb2db8. Removed in https://github.com/jchavarri/dune/pull/4/commits/129567ffa3a6cef42935ae34c21614c70cdd5f87.